.ci/setenvconfig

```diff
@@ -127,6 +124,19 @@ CLUSTER_NAME=kind-$BUILD_NUMBER
 TEST_OPTS=-race
 E2E_TEST_ENV_TAGS="kind,$4"
 ENV
+write_deployer_config <<CFG
+id: kind-ci
+overrides:
+  clusterName: eck-e2e-$BUILD_NUMBER
+  clientVersion: $3
+  vaultInfo:
+    address: $VAULT_ADDR
+    roleId: $VAULT_ROLE_ID
+    secretId: $VAULT_SECRET_ID
+  kind:
+    nodeImage: $2
+    ipFamily: $4
```

Comment:
Nit: keep the variables indirection to clearly explicit the arguments required to call `e2e/kind-k8s-version`.
```sh
  e2e/kind-k8s-versions)
######################################

kindVersion=$2
kindNodeImage=$3
ipFamily=$4

write_env <<ENV
...
E2E_TEST_ENV_TAGS="kind,$ipFamily"   // use
ENV
write_deployer_config <<CFG
id: kind-ci
overrides:
  clusterName: eck-e2e-$BUILD_NUMBER
  clientVersion: ${kindVersion}
  vaultInfo:
    address: $VAULT_ADDR
    roleId: $VAULT_ROLE_ID
    secretId: $VAULT_SECRET_ID
  kind:
    nodeImage: ${kindNodeImage}
    ipFamily: ${ipFamily}
CFG
;;
```